govcms_ckan.module

```diff
@@ -158,6 +164,35 @@ function govcms_ckan_get_plugin_config($name, $form, $form_state, $config) {
 }
 
 /**
+ * Helper function to return a specific visualisation plugin config form.
+ *
+ * @param string $name
+ *   The name of the visualisation plugin to return.
+ * @param array $file
```

Comment:
Object :) 
Also the description for that comment was wrong, returns view not form. Both fixed in #14